src/SessionMonitor.ts

```diff
@@ -153,13 +153,8 @@ export class SessionMonitor {
                     raiseEvent = false;
                     this._checkSessionIFrame.start(session.session_state);
 
-                    if (session.sid === this._sid) {
```

Comment:
i do not understand why this should work:
- We still use session.sub at line 152
- With your changes here i would expect that `_raiseUserSessionChanged` is called all the time when the user is still ok...

Reply:
lets keep the `sub` claim